common/templates/common/403.html

```diff
@@ -0,0 +1,19 @@
+{% extends "common/base.html" %}
+
+{% block pageTitle %}
+Sorry, you don't currently have permission to access this service – {{ service_name }} – GOV.UK
+{% endblock pageTitle %}
+
+{% set mainClasses = "govuk-main-wrapper--l" %}
+
+{% block content %}
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-l">Sorry, you don't currently have permission to access this service</h1>
+    <p class="govuk-body">
+      If you believe this is incorrect, please email us on
+      <a href="mailto:{{ config['CONTACT_EMAIL'] }}">{{ config['CONTACT_EMAIL'] }}</a>
```

Comment:
tiny nit just to fix some styling (same in a few other places) 🙏 

```suggestion
      <a class="govuk-link" href="mailto:{{ config['CONTACT_EMAIL'] }}">{{ config['CONTACT_EMAIL'] }}</a>
```

Reply:
Cheers Sam, nice catch, have updated on the common templates 👍